test/playlists.test.js

```diff
@@ -0,0 +1,68 @@
+import QUnit from 'qunit';
+import { makeMochTech } from './util/util.js';
+import sinon from 'sinon';
+import { FlashlsHandler } from '../src/index.js';
+
+QUnit.module('Flashls playlists', {
```

Comment:
your test doesnt use timers for anything so you can just make this `QUnit.module('Flashls playlists');` and just remove sinon from this file completely